src/commonMain/kotlin/exchange.dydx.abacus/state/v2/supervisor/ConnectionStats.kt

```diff
@@ -245,32 +246,36 @@ internal class ConnectionStats(
         updateApiState()
     }
 
+    private fun apiStateParams(): IMap<String, Any>? {
+        val indexerTime = lastIndexerCallTime?.toEpochMilliseconds()?.toDouble()
+        val validatorTime = lastValidatorCallTime?.toEpochMilliseconds()?.toDouble()
+        val interval = if (indexerTime != null) {
+            (
+                Clock.System.now().toEpochMilliseconds()
+                    .toDouble() - indexerTime
```

Comment:
Why are we using Doubles for epoch milliseconds? Shouldn't we just keep them as Longs?

Reply:
errr not sure if there's reason behind it -- can switch to Long here during the subtraction but maybe need to keep them as Doubles when sending to amplitude for consistency?